envconfig/config.go

```diff
@@ -131,10 +137,11 @@ func clean(key string) string {
 
 func init() {
 	// default values
-	NumParallel = 0 // Autoselect
+	NumParallel = 8 // Autoselect
```

Comment:
Why?

Reply:
This change is not needed.
Default can be set to 0